### PR TITLE
fix: Relative time link on BSX

### LINKED ISSUE #3280: Relative time link on BSX doesn't work
Currently, links from tooltips on relative time (history component) are leading nowhere (https://basilisk.subscan.io/block/55762). Let's remove the link for BSX for now

image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3280
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

File: components/rmrk/Gallery/History.vue [hunk 1]
```diff
@@ -244,6 +246,11 @@ export default class History extends mixins(
     return [HistoryEventType.ALL, Interaction.BUY].includes(this.event)
   }
 
+  get disableBlockUrl() {
```
Comment (roiLeo) [inline components/rmrk/Gallery/History.vue:249]: proper naming & return type

```suggestion
  get hasDisabledBlockUrl(): boolean {
```

**Review by roiLeo — COMMENTED** [review]
small stuff

- would be cool to create a new component `BlockExplorerLink` in the future
- it's starting to get messy with the components path: `/rmrk/Gallery/History.vue` while it's being used on other chains

**Review by roiLeo — APPROVED** [review]
✅ code lgtm
❌ preview deploy

**Comment by Jarsen136** [comment]
> random note:
> 
> * would be cool to create a new component `BlockExplorerLink` in the future
> * it's starting to get messy with the components path: `/rmrk/Gallery/History.vue` while it's being used on other chains

I agree with you 👍

**Comment by yangwao** [comment]
> random note:
> - would be cool to create a new component `BlockExplorerLink` in the future
> - it's starting to get messy with the components path: `/rmrk/Gallery/History.vue` while it's being used on other chains

@petersopko let's make follow up issues on these two :)

**Comment by Jarsen136** [comment]
✅  preview deployed

**Comment by Jarsen136** [comment]
> ✅ preview deployed

image

Deploy failed may come from this commit.

**Comment by petersopko** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 51.55 USD/KSM ~ 0.97 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xcc2036a8c0fb0bc96d02766a0a6005256acbe03e2ec35eb28841876c4d2bb25c](https://kusama.subscan.io/extrinsic/0xcc2036a8c0fb0bc96d02766a0a6005256acbe03e2ec35eb28841876c4d2bb25c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**